### ISSUE TITLE
Cakeshop not working in Kubernetes configurations.

### ISSUE THREAD

**Issue by melihbirim** [ConsenSys/quorum-wizard#132]
### Generate a network with configurations below.

```
? Would you like to generate bash scripts, a docker-compose file, or a kubernetes config to bring up your network? kubernetes
? Select your consensus mode - istanbul is a pbft inspired algorithm with transaction finality while raft provides faster blocktimes, transaction finality and o
n-demand block creation raft
? Input the number of nodes (2-7) you would like in your network - a minimum of 3 is recommended 3
? Which version of Quorum would you like to use? Quorum 21.1.0
? Choose a version of tessera if you would like to use private transactions in your network, otherwise choose "none" Tessera 21.1.0
? What tools would you like to deploy alongside your network? (Press space to select options, then press enter) Cakeshop, Quorum's official block explorer
? Would you like to generate keys for your network? (selecting 'no' will use insecure keys that are not suitable for Production use) No
? 10 is the default network id in quorum but you can use a different one 566
? What would you like to call this network? test-cakeshop-network
```

Here is the logs fro kubectl after start.sh 

```
kubectl  logs cakeshop-deployment-764887f8b6-dvnsn
+ cp /etc/cakeshop/cakeshop-nodes.json /cakeshop/cakeshop-nodes.json
+ sed -i s/QUORUM_NODE1_SERVICE_HOST/10.106.231.164/g /cakeshop/cakeshop-nodes.json
+ sed -i s/QUORUM_NODE2_SERVICE_HOST/10.101.149.218/g /cakeshop/cakeshop-nodes.json
+ sed -i s/QUORUM_NODE3_SERVICE_HOST/10.106.12.64/g /cakeshop/cakeshop-nodes.json
+ /bin/tini -- /usr/local/bin/cakeshop.sh
sh: /bin/tini: not found
```

The expected outcome would be running cakeshop instead of the output above.

**Comment by cdesch** [comment]
Hit the same issue. Any luck on what it is?

**Comment by cdesch** [comment]
@melihbirim changing the `cakeshop` image version to `image: quorumengineering/cakeshop:0.11.0` fixes this. 

I made some notes in [this issue](https://github.com/ConsenSys/cakeshop/issues/207)

**Comment by bmcd** [comment]
Looks like updating the qubernetes project to work with cakeshop 0.12.x will be pretty easy. Then we can update wizard and the issue will be fixed.

Thanks to you both for reporting the issue and I'll keep you updated on progress!

**Comment by cdesch** [comment]
thanks @bmcd

**Comment by melihbirim** [comment]
@cdesch, @bmcd has already opened a PR, waiting it to be accepted to use. Thanks for the fix.

**Comment by cdesch** [comment]
Awesome!

**Comment by bmcd** [comment]
Fixed in 1.3.2! Thanks again for pointing out the issue

**Comment by cdesch** [comment]
@bmcd you're welcome
